### PR TITLE
Add clarification for portable command line options behaviour oddity

### PR DESCRIPTION
**Pull request by CS-Account**
The documentation for command line options stated that 2 options `--user-data-dir <dir>` and `--extensions-dir <dir>` did not work in Portable Mode, though this was not entirely accurate as the features were neither deprecates nor nonfunctional but did have special conditions for usage (they need no data folder to be present).

Related to this, some documentation made it unclear that `Microsoft VS Code/bin/code` was the recommended executable from which to launch vscode for command line arguments to function.

[Relevant issue with more detail](https://github.com/microsoft/vscode/issues/181876)

### CODE REVIEW

**Comment by CS-Account** [comment]
@microsoft-github-policy-service agree

File: docs/editor/command-line.md [hunk 1]
```diff
@@ -102,8 +102,8 @@ There are several CLI options that help with reproducing errors and advanced set
 
 Argument|Description
 ------------------|-----------
-`--extensions-dir <dir>` | Set the root path for extensions. Has no effect in [Portable Mode](/docs/editor/portable.md).
-`--user-data-dir <dir>` | Specifies the directory that user data is kept in, useful when running as root. Has no effect in [Portable Mode](/docs/editor/portable.md).
+`--extensions-dir <dir>` | Set the root path for extensions. (Overridden in [Portable Mode](/docs/editor/portable.md) by the presence of the data folder\*).
```
Comment (cadamini) [inline docs/editor/command-line.md:105]: Interesting details in the description of the [original commit](https://github.com/microsoft/vscode-docs/commit/fe4fbc1398a12b5698f836e84d7a1fec2abb56a9). 

> Portable mode (presence of  `/data/` folder in VSC installation directory) effectively prevents `--user-data-dir` and `--extensions-dir` switches from working.

```suggestion
`--extensions-dir <dir>` | Set the root path for extensions. (Overridden in [Portable Mode](/docs/editor/portable.md) by the presence of the data folder in the VS Code installation).
```

File: docs/editor/command-line.md [hunk 2]
```diff
@@ -102,8 +102,8 @@ There are several CLI options that help with reproducing errors and advanced set
...
+`--user-data-dir <dir>` | Specifies the directory that user data is kept in, useful when running as root. (Overridden in [Portable Mode](/docs/editor/portable.md) by the presence of the data folder\*).
```
Comment (cadamini) [inline docs/editor/command-line.md:106]: Also, not sure about the asterisk.

```suggestion
`--user-data-dir <dir>` | Specifies the directory that user data is kept in, useful when running as root. (Overridden in [Portable Mode](/docs/editor/portable.md) by the presence of the data folder in the VS Code installation).
```

**Review by cadamini — APPROVED** [review]
LGTM but see my comments

**Review by cadamini — APPROVED** [review]

**Review by cadamini — APPROVED** [review]

**Review by gregvanl — APPROVED** [review]
Thanks

**Pull request merged**